### ISSUE TITLE
Installed to Wii U, getting "No fat device found" error

### ISSUE THREAD

**Issue by urko948** [FIX94/Nintendont#996]
When running USB Loader GX, I can load my Wii backup discs just fine through it. But whenever I try to load Nintendon't stand alone under HB Launcher or if I'm opening a gamecube game through USB Loader GX, I get that error.

I'm using a WD 2 TB external drive formatted to FAT32 and I am using a Y cable. I tried reinstalling it and deleting my other copy of Nintendon't and using this version, but can't get it working no matter what I have tried. Anyone have any ideas? Thank you in advance.

**Comment by th-2021** [comment]
How did you format the drive? I had the same issue, but mine got resolved by formatting with Linux (mkfs.vfat).

**Comment by jordanallen291** [comment]
I had a similar problem. I was using TeconMoon's WiiVC Injector to inject Wii and Gamecube games as Wii U VC titles as I wanted all my games on one drive. From there I used USB Loader GX to install to the games my WD Elements (4 TB formatted to FAT32, then formatted by the Wii U - my dad gave me one from his work at IT, I know I'm not going to fill the whole thing up).

Wii injects worked fine, but GC games gave me the same, "No fat device found."

After hours of troubleshooting, what finally worked for me was putting an SD card in the front (FAT32). It DOES NOT need to have the game data on the card though. I installed Ocarina of Time/Master Quest to my HDD and, as long as I have an SD card in front, it was working. That being said, that SD card has my Tiramisu loadout with nintendont on it. I'm not entirely sure why this works (more troubleshooting will reveal that), but my initial (very naive) thoughts are that nintendont doesn't recognize the HDD as a viable place to store and save data for the games. I'll do some more testing, but this solution may work for you with some trial and error in what files are needed on your SD card.

tl;dr: Put a CFW SD card in the front (No game installation data needed).

**Comment by carnage702** [comment]
> I had a similar problem. I was using TeconMoon's WiiVC Injector to inject Wii and Gamecube games as Wii U VC titles as I wanted all my games on one drive. From there I used USB Loader GX to install to the games my WD Elements (4 TB formatted to FAT32, then formatted by the Wii U - my dad gave me one from his work at IT, I know I'm not going to fill the whole thing up).
> 
> Wii injects worked fine, but GC games gave me the same, "No fat device found."
> 
> After hours of troubleshooting, what finally worked for me was putting an SD card in the front (FAT32). It DOES NOT need to have the game data on the card though. I installed Ocarina of Time/Master Quest to my HDD and, as long as I have an SD card in front, it was working. That being said, that SD card has my Tiramisu loadout with nintendont on it. I'm not entirely sure why this works (more troubleshooting will reveal that), but my initial (very naive) thoughts are that nintendont doesn't recognize the HDD as a viable place to store and save data for the games. I'll do some more testing, but this solution may work for you with some trial and error in what files are needed on your SD card.
> 
> tl;dr: Put a CFW SD card in the front (No game installation data needed).

Obviously you need an SD card, nintendont can't save save games or nintendont settings on Wiiu formatted hdds

**Comment by jordanallen291** [comment]
That makes sense. After doing some more trouble shooting I think what the problem might have been was that I had a micro-SD card inserted into an adapter to upload the games to the HDD. This was a Lexar PLAY 1TB microSDXC UHS-I-Card (Formatted to FAT32). When I had this card in the front, I received the, "no fat device found" error, but when I put in a normal 32 gb SanDisk, everything started working. I'm not sure which of these factors caused the error, but that's how I fixed the problem.

**Comment by carnage702** [comment]
> That makes sense. After doing some more trouble shooting I think what the problem might have been was that I had a micro-SD card inserted into an adapter to upload the games to the HDD. This was a Lexar PLAY 1TB microSDXC UHS-I-Card (Formatted to FAT32). When I had this card in the front, I received the, "no fat device found" error, but when I put in a normal 32 gb SanDisk, everything started working. I'm not sure which of these factors caused the error, but that's how I fixed the problem.

i use a micro sd card with adapter on wiiu since day one, sometimes the cards dont connect right you need to take them out and put the micro back in and then back into the wiiu, several times they dont even display right on homebrew channel/launcher.
